cms/envs/common.py

```diff
@@ -793,6 +793,7 @@
 
 OPTIONAL_APPS = (
     'mentoring',
+    'problem_builder',
```

Comment:
@bradenmacdonald You will also need to add it to https://github.com/edx/edx-platform/blob/97f0cf108f6bd96ad678c6b3a168163f179e3e7d/cms/envs/common.py#L856 to allow it to be addable to a unit, this only allows to get the migrations to run.